docs/developer-guide/aggregate-streaming-data.rst

```diff
@@ -0,0 +1,110 @@
+.. _aggregate-streaming-data-with-ksql:
+
+Aggregate Streaming Data With KSQL
+##################################
+
+KSQL supports several aggregate functions, like COUNT and SUM. You can use
+these to build stateful aggregates on streaming data. For the full list, see
+:ref:`ksql_aggregate_functions`.
+
+Here are some examples that show how to aggregate data from an inbound stream
+of pageview records, named ``pageviews``.
+
+Aggregate Results in a KSQL Table

```

Comment:
Do we already have an animated gif that we could include here?

For the time being, we could re-use the one I created at http://www.michael-noll.com/assets/uploads/stream-table-animation-numVisitedLocations.gif ([article](http://www.michael-noll.com/blog/2018/04/05/of-stream-and-tables-in-kafka-and-stream-processing-part1/)), which is to demonstrate how COUNT() works.